### PR TITLE
Replace Videokonferenz with Notensystem on Website

### LINKED ISSUE #1555: Replace "Videokonferenzen" with Grades feature on our website

### PR DESCRIPTION
**Pull request by EvilMonkey09**
Fixes #1555

### CODE REVIEW

File: website/lib/home/src/all_in_one_platform.dart [hunk 1]
```diff
@@ -96,13 +96,14 @@ class AllInOnePlaceState extends State<AllInOnePlace> {
...
-                          "Hosting in Deutschland"
-                        ]),
-                        feature(
-                          "Immer verfügbar",
-                          bulletpoints: [
+                        feature("Notensystem", bulletpoints: [
+                          "Speichere deine Noten in Sharezone",
+                          "Verschiedene Notensysteme"
+                          ],
+                          leaveDefaultPicture: true,
```
Comment (Jonas-Sander) [inline website/lib/home/src/all_in_one_platform.dart:103]:
```suggestion
```

That has to be deleted, in order for it to work

**Review by Jonas-Sander — APPROVED** [review]
LGTM. Seems like you need to format the dart file again. Also consider merging origin/main into this branch again.

**Review by nilsreichardt — APPROVED** [review]
LGTM

**Comment by Jonas-Sander** [comment]
Thanks 🩵

**Pull request merged**